docs/guides/operator-private-gcp-confidential-space.md

```diff
@@ -0,0 +1,405 @@
+
+   ```
+   http://{PUBLIC_IP}/ops/healthcheck
+   ```
+
+#### Clean Up
+
+Remove all resources created by Terraform:
+
+```
+terraform destroy
+```
+
+#### Inputs
+
+(**GWH_YS18 I need more data re the inputs and outputs so we can have some little intro line to these tables. Is this something they clean up? Or something that's required? Please help with some explanation for both inputs and outputs.**)
+
+| Name               | Description | Type     | Default             | Required |
+|--------------------|-------------|----------|---------------------|:--------:|
+| project_id         | n/a         | `string` | n/a                 |   yes    |
+| service_account    | n/a         | `string` | n/a                 |   yes    |
+| uid_operator_image | n/a         | `string` | n/a                 |   yes    |
+| uid_api_token      | n/a         | `string` | n/a                 |   yes    |
+| region             | n/a         | `string` | `"asia-southeast1"` |    no    |
+| network_name       | n/a         | `string` | `"uid-operator"`    |    no    |
+| uid_machine_type   | n/a         | `string` | `"n2d-standard-16"` |    no    |
+| uid_deployment_env | n/a         | `string` | `"integ"`           |    no    |
+| max_replicas       | n/a         | `number` | `5`                 |    no    |
+| min_replicas       | n/a         | `number` | `1`                 |    no    |
+| debug_mode         | n/a         | `bool`   | `false`             |    no    |
+
+#### Outputs
+
+| Name             | Description |
+|------------------|-------------|
+| load-balancer-ip | n/a         |
+
+#### Terraform Template&#8212;Notes
+
+1. You might want to change the load balancer from HTTP to HTTPS. To do this, follow these steps: (**GWH_YS19 surely they would have to use HTTPS? Should this be higher profile than a note at the end?**)
+
+   - Provide your cert via terraform following this page:
+  [google_compute_ssl_certificate](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_ssl_certificate.html)
+
+   - Then add the following configuration values in `module "gce-lb-http"` (**GWH_YS20 is the syntax correct here? Is it a name/value pair? Don't understand this. Module not previously mentioned. It's a module of what?**)
+
+     ```
+       ssl                  = true
+       ssl_certificates     = [google_compute_ssl_certificate.you_cert.self_link]
+       use_ssl_certificates = true
+       https_redirect       = true
+     ```
+
+2. Follow the [Prerequisites](https://github.com/IABTechLab/uid2-operator/tree/master/scripts/gcp-oidc#prerequisites)
+   section to set up a service account. (**GWH_YS21 this seems a little unkind to make the last step something they still need to do. Is this really all the information needed? Will it all make sense to our users and be easy to follow and implement?**)
```

Comment:
Sry my readme is not well-organized, we shoud move it to the top, before they run anything.

Ideally we could also provisioning service_account via terraform template, no bandwidth yet. 

The template file and requirements here is subject to change & iterate later. We could come up with an init version first.